npm-shrinkwrap.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fxa-content-server",
-  "version": "1.120.0",
+  "version": "1.119.3",
```

Comment:
This is strange, it should not have changed to `1.119.3`. Mind rebasing this branch with our master branch? You might have to also do an `npm install` after rebasing.